.github/workflows/build.yml

```diff
@@ -42,3 +42,28 @@ jobs:
 
       - name: Run Tests
         run: ./gradlew clean check -x detekt
+
+  pluginVerifier:
+    name: Plugin Verifier
+    needs: gradleValidation
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Fetch Sources
+        uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

Comment:
Why is java-version 11 here, whereas we test with 1.8 version in the test job? Can we leave an inline comment perhaps or is it me not knowing the verification specifics coming from a different tech stack?

Reply:
Good question @michelkaporin!. We need Java 11 because of this error: Error: 

`Exception in thread "main" java.lang.UnsupportedClassVersionError: **_com/jetbrains/pluginverifier/PluginVerifierMain has been compiled by a more recent version of the Java Runtime_** (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0`.

Reply:
I see, can you perhaps leave a comment with explanation for this? Will be valuable in future, when we forget 👍

Reply:
@michelkaporin added comment for this ;). Thanks! 👍